src/components/views/EResources.js

```diff
@@ -313,6 +313,8 @@ export default class EResources extends React.Component {
                       onNeedMoreData={onNeedMoreData}
                       isSelected={({ item }) => item.id === selectedRecordId}
                       onRowClick={onSelectRow}
+                      pageAmount={100}
```

Comment:
If this maps to the RESULT_COUNT_INCREMENT in EResourcesRoute, can we move that constant to `constants/` and import it from there in here and EResourcesRoute?